src/app/Http/Controllers/Operations/InlineCreateOperation.php

```diff
@@ -19,20 +19,31 @@ protected function setupInlineCreateRoutes($segment, $routeName, $controller)
         Route::post($segment.'/inline/create/modal', [
             'as'        => $segment.'-inline-create',
             'uses'      => $controller.'@getInlineCreateModal',
-            'operation' => 'InlineCreate',
+            'operation' => 'DefaultInlineCreate',
```

Comment:
Any chance we can use the name of the method `setupDefaultInlineCreateOperation` without renaming the operation itself from `InlineCreate` to `DefaultInlineCreate`? I'm thinking that people might add stuff to the `InlineCreate` operation (🤷‍♂️) so what use in renaming it?